dist/lib/js/components/PopOver/PopOver.consts.d.ts

```diff
@@ -12,3 +12,7 @@ export declare const POP_OVER_PLACEMENTS: {
     readonly LEFT: "left";
     readonly RIGHT: "right";
 };
+export declare const POP_OVER_SIZES: {
```

Comment:
@kvas-damian I assume that changes in this file shouldn't be commited, right?

Reply:
yes, you shoudn't be commiting any files in `dist/` directory

Reply:
Done